imaging.yaml

```diff
@@ -3,125 +3,158 @@ install_resolver_dependencies: true
+    
+  - name: anisotropic_diffusion
     owner: imgteam
     tool_panel_section_label: "Imaging"
+    
+  - name: imagej2_find_maxima
+    owner: iuc
+    tool_panel_section_label: "Imaging"
+    tool_shed_url: https://testtoolshed.g2.bx.psu.edu/
 
-  - name: 2d_simple_filter
-    owner: imgteam
+  - name: imagej2_make_binary
+    owner: iuc
     tool_panel_section_label: "Imaging"
+    tool_shed_url: https://testtoolshed.g2.bx.psu.edu/  
 
-  - name: anisotropic_diffusion
+  - name: 2d_auto_threshold
     owner: imgteam
     tool_panel_section_label: "Imaging"
-
-  - name: binary2labelimage
+    
+  - name: imagej2_watershed_binary
+    owner: iuc
+    tool_panel_section_label: "Imaging"
+    tool_shed_url: https://testtoolshed.g2.bx.psu.edu/
+            
+  - name: landmark_registration
     owner: imgteam
     tool_panel_section_label: "Imaging"
-
-  - name: binaryimage2points
+    
+  - name: projective_transformation
     owner: imgteam
     tool_panel_section_label: "Imaging"
 
-  - name: colocalization_viz
+  - name: projective_transformation_points
     owner: imgteam
     tool_panel_section_label: "Imaging"
+    
+  - name: imagej2_math
+    owner: iuc
+    tool_panel_section_label: "Imaging"
+    tool_shed_url: https://testtoolshed.g2.bx.psu.edu/
 
-  - name: color_deconvolution
+  - name: imagecoordinates_flipaxis
     owner: imgteam
     tool_panel_section_label: "Imaging"
 
-  - name: concat_channels
+  - name: binary2labelimage
     owner: imgteam
     tool_panel_section_label: "Imaging"
 
-  - name: coordinates_of_roi
+  - name: binaryimage2points
     owner: imgteam
     tool_panel_section_label: "Imaging"
 
-  - name: count_objects
+  - name: mergeneighboursinlabelimage
     owner: imgteam
     tool_panel_section_label: "Imaging"
 
-  - name: curl_post
+  - name: scale_image
     owner: imgteam
-    tool_panel_section_label: "Send Data"
+    tool_panel_section_label: "Imaging"
 
-  - name: detection_viz
+  - name: slice_image
     owner: imgteam
     tool_panel_section_label: "Imaging"
 
-  - name: labelimage2points
+  - name: split_labelmap
     owner: imgteam
     tool_panel_section_label: "Imaging"
 
-  - name: mahotas_features
+  - name: concat_channels
     owner: imgteam
     tool_panel_section_label: "Imaging"
-
-  - name: mergeneighboursinlabelimage
+    
+  - name: points2binaryimage
     owner: imgteam
     tool_panel_section_label: "Imaging"
-
-  - name: overlay_moving_and_fixed_image
+    
+  - name: points2labelimage
     owner: imgteam
     tool_panel_section_label: "Imaging"
-
-  - name: overlay_segmentation_mask
+    
+  - name: labelimage2points
     owner: imgteam
     tool_panel_section_label: "Imaging"
-
+    
   - name: permutate_axis
     owner: imgteam
     tool_panel_section_label: "Imaging"
-
-  - name: points2binaryimage
+    
+  - name: imagej2_analyze_skeleton
+    owner: iuc
+    tool_panel_section_label: "Imaging"
+    tool_shed_url: https://testtoolshed.g2.bx.psu.edu/
```

Comment:
@ThomasWollmann we can not add tools from the TestToolShed. Have you tried those tools? Do you think they are ready?

@gregvonkuster are you interested in updating those?

Reply:
@bgruening Do you mean all of the imagej2 tools that are on the TTS?  It looks like we never moved these to the MTS (at least they are not available there now), but I don't remember why.

Was it due to the required tweak to the local job runner (IIRC, this line https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/jobs/runners/local.py#L101 needs to be commented out) that is required for the tools to function so that CI tests can pass?  

It's been a long time since I've worked with these tools, so has the test process for the tools-idc repo changed in some way to allow these tools to now be tested without tweaking the local job runner?

Reply:
I think this was the issue yes. I do not know if this is fixed, but many things have changed, uwsgi and so on, so I just hope it is fixed :)

Reply:
@bgruening  I tried them a while ago and they seemed fine. If we can not add them I have to reimplement them, since they perform basic tasks that are essential for many image analysis tasks.

Reply:
@ThomasWollmann I think the first thing to try is to see if one of the tools (any one you choose should do) will function with the local job runner without making any changes to the runner code.  I can do this, just haven't had time yet.  If it turns our that the tool works without changing the job runner, then we can probably upload them to the MTS.

The reason that they were not working in the past is because the tools use ImageJ2 (i.e., Fiji https://imagej.net/ImageJ2) which runs a JVM, and the JVM caused problems with this line in the local job runner https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/jobs/runners/local.py#L101, resulting in the tool not being able to finish.

Reply:
@ThomasWollmann I'm all in to add those packages, I'm just wondering if we should update them before. They are now a few years old.

@gregvonkuster do you want to give it a try?

Reply:
I can certainly do this, but it will be a bit before I can get to it.  If you'd like it sooner, I can help out with advice.  I can get started fairly soon, but will have to work on them intermittently between the corals work I'm doing.

Reply:
@bgruening It's been so long since I've worked with your galaxytools repo that I've hit the following issue.  I'm doing this on osx.  No matter what I try, I cannot get around this file name case issue.  Do you know a way to fix this beyond deleting my repo and re-forking yours?

```
$ git clone https://github.com/gregvonkuster/galaxytools.git
Cloning into 'galaxytools'...
remote: Enumerating objects: 25401, done.
remote: Total 25401 (delta 0), reused 0 (delta 0), pack-reused 25401
Receiving objects: 100% (25401/25401), 126.67 MiB | 3.09 MiB/s, done.
Resolving deltas: 100% (15317/15317), done.
Checking out files: 100% (2743/2743), done.
warning: the following paths have collided (e.g. case-sensitive paths
on a case-insensitive filesystem) and only one from the same
colliding group is in the working tree:

  'chemicaltoolbox/chemfp/test-data/CID_2244_MACCS.fps'
  'chemicaltoolbox/chemfp/test-data/CID_2244_maccs.fps'
$ git --version
git version 2.20.1 (Apple Git-117)```

Reply:
Uha, yes I have seen this. But I thought I fixed that ... maybe it is in your history ... Oo, can you try to clone it into another repo and see if it works.

Reply:
This is a fresh clone, of my fork, of course.  Nothing I've tried works, so I guess I should delete my fork?

Reply:
Can you clone from upstream and see if this works before you delete your fork?

Reply:
I cloned your repo cleanly as an origin.  What would I do next?

Reply:
Uh, more roadblocks!  :(   I'm running the tip of the galaxy dev branch and I'm having no success even installing a tool from the TTS.  Is the TS api install process not working?

Reply:
Ah, I found this in the log, so looks like I need to fix something in my Python env.
```
returned non-zero exit status 255\nOutput was:\nabort: error: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed (_ssl.c:727)
```

Reply:
I think the TTS is nowadays not activated, so you should activate it, but ... why are you not using planemo? Why are you installing from the TTS?

Reply:
I was going to see whether the local job runner would now work for the tools after installation, but I guess we can justifiably assume it still won't.  So what are you looking for me to do here?  Do you want me to just update the Fiji bioconda recipe, planemo test, and upload to the MTS?

Reply:
Just an FYI - I'm working on the bioconda recipe for the latest Fiji (20170530).  I've got most of it working, but the bunwarpj Java stuff has somehow changed, so that test is not working.  I'll have to make a look at what has changed.  Hopefully won't take too long.

Reply:
I had hoped that this is working now with the local job runner - and with this with plain planemo. I think what we need is updated wrappers :)

Thanks for working on conda, thats a good first step!! Thanks!!

Reply:
@gregvonkuster thank you for keeping up the work! Would be nice if we don't have to reinvent the wheel.

Reply:
@gregvonkuster Some tools are only available through ImageJ. However starting the JVM is relatively slow. For some tasks I already implemented tools based on scikit-image: https://toolshed.g2.bx.psu.edu/repository/browse_repositories_in_category?status=done&message=&id=f32bb8f087d8a30d